### PR TITLE
Update docs-tech-guide.rst

### PR DESCRIPTION
**Pull request by AliceUmeji**
#### What is included in this PR?

Added a tip for making edits directly in github.

### CODE REVIEW

File: odkx-src/docs-tech-guide.rst [hunk 1]
```diff
@@ -1067,8 +1067,16 @@ Working on the docs
...
-
+   - :doc:`docs-developer-guide`.
+   	
+    	.. tip::
+		
+		You can make minor edits to files directly on github. To do that:
+			#. Browse and open the file you want to edit
+			#. Click the pencil icon at the top right corner in order to edit the file
+			#. Make your changes
+			#. Scroll to the bottom of the page to commit your changes
```
Comment (elmps2018) [inline odkx-src/docs-tech-guide.rst:1078]: Can you add a screen shot? Or link to directions that explain/video this more generally?

**Review by elmps2018 — COMMENTED** [review]
@AliceUmeji thanks for working on this! Some suggestions to make it clearer.

File: odkx-src/docs-tech-guide.rst [hunk 2]
```diff
@@ -1067,8 +1067,16 @@ Working on the docs
...
 
-   - :doc:`docs-developer-guide`
-
+   - :doc:`docs-developer-guide`.
+   	
+    	.. tip::
+		
+		You can make minor edits to files directly on github. To do that:
+			#. Browse and open the file you want to edit
+			#. Click the pencil icon at the top right corner in order to edit the file
```
[the remark this answers is not in the document]
Comment (AliceUmeji) [reply odkx-src/docs-tech-guide.rst:1076]: Ok. I'll make the changes and add a link.

File: odkx-src/docs-tech-guide.rst [hunk 3]
```diff
@@ -1071,12 +1071,14 @@ Working on the docs
    	
     	.. tip::
 		
-		You can make minor edits to files directly on github. To do that:
-			#. Browse and open the file you want to edit
+		You can make minor edits to files directly on github. To do this:
+			#. Browse the folder in which the file to be edited is located
```
Comment (elmps2018) [inline odkx-src/docs-tech-guide.rst:1075]: Browse to the folder, to is missing

**Review by elmps2018 — APPROVED** [review]

**Review by elmps2018 — APPROVED** [review]

**Pull request merged**